File: Proyecto/js/Usuario.js
 // funcion Usuario
function Usuario (cedula,usuario,contraseña,nombre,apellidos,email,telefono,celular,fecha){ 
  this.Usuario(cedula,usuario,contraseña,nombre,apellidos,email,telefono,celular,fecha);
}
function Usuario(usuario) {
  this.Usuario1(usuario);
}

// solo metodos
Usuario.prototype={ 
  Usuario: function(cedula,usuario,contraseña,nombre,apellidos,email,telefono,celular,fecha){
    this.cedula=cedula;
    this.usuario= usuario;
    this.contraseña=contraseña;
    this.nombre=nombre;
    this.apellidos=apellidos;
    this.email=email;
    this.telefono= telefono;
    this.celular=celular;
    this.fecha=fecha;
  },
  Usuario1: function(usuario) {
    this.usuario=usuario;
  }
}

File: Progra4_project/src/java/AAMAirline/model/Usuario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package AAMAirline.model;

/**
 *
 * @author jimen
 */
public class Usuario implements Jsonable{

    public Usuario(String cedula, String usuario, String contraseña, String nombre, String apellidos, String email, String telefono, String celular, String fecha) {
        this.cedula = cedula;
        this.usuario = usuario;
        this.contraseña = contraseña;
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.email = email;
        this.telefono = telefono;
        this.celular = celular;
        this.fecha = fecha;
    }

    public String getCedula() {
        return cedula;
    }

    public void setCedula(String cedula) {
        this.cedula = cedula;
    }

    public String getUsuario() {
        return usuario;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public String getContraseña() {
        return contraseña;
    }

    public void setContraseña(String contraseña) {
        this.contraseña = contraseña;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getCelular() {
        return celular;
    }

    public void setCelular(String celular) {
        this.celular = celular;
    }

    public String getFecha() {
        return fecha;
    }

    public void setFecha(String fecha) {
        this.fecha = fecha;
    }
    
    private String cedula;
    private String usuario;
    private String contraseña;
    private String nombre;
    private String apellidos;
    private String email;
    private String telefono;
    private String celular;
    private String fecha;
}
